Features5/tests/features/newUserRegistration.feature

```diff
@@ -0,0 +1,28 @@
+Feature:
+   As a user
+   I want to register
+   So that I can use the contact fox app
+
+   Background:
+       Given user has been navigated to registration page
+   
+   Scenario: New user registration
+       When user enters following valid credentials:
+         |name    |email                |password   |confirmPassword  |
+         |Susmita |susmita412@gmail.com |zxcvbnm    |zxcvbnm          |
+         |Sita    |sita123@gmail.com    |abcdef     |abcdef           |
+       And clicks "Register" button
```

Comment:
The `register` button is clicked after filling both the user's data. How would the first user be registered?